libcontainer/console_linux.go

```diff
@@ -89,7 +89,14 @@ func (c *linuxConsole) mount(rootfs, mountLabel string, uid, gid int) error {
 	if f != nil {
 		f.Close()
 	}
-	return syscall.Mount(c.slavePath, dest, "bind", syscall.MS_BIND, "")
+	if err := syscall.Mount(c.slavePath, dest, "bind", syscall.MS_BIND, ""); err != nil {
+		return err
+	}
+
+	if err := syscall.Mount("", dest, "", syscall.MS_PRIVATE, ""); err != nil {
+		return err
```

Comment:
I would rather keep it this way. If making it PRIVATE failed, we need to know why it failed. I don't think this is warning thing.